server/routerlicious/packages/lambdas/src/scribe/checkpointManager.ts

```diff
@@ -82,6 +83,7 @@ export class CheckpointManager implements ICheckpointManager {
 						this.documentId,
 						expectedSequenceNumber - 1,
 						expectedSequenceNumber + 1,
```

Comment:
could we do some guard/protection about undefined/null value here too? Your fix on get delta level is neat and catch all too. It seems the bug are at scribe level too and might affect other flows so it would be great that if we could fix the bug here too.

Reply:
Can you share more about the scribe bug? I think its better to handle that separately and keep this PR scoped to get deltas API.

Reply:
It is the large logtail mentioned by @pradeepvairamani in the OCE hand off meeting

Reply:
<span>Execute in [<a href="https://dataexplorer.azure.com/clusters/frspme.westus2/databases/FRS?query=H4sIAAAAAAAAA5WQTUvDQBCG7/0Vw54SCMZ6kkK8CKI3rd5KkdWMyZbshzMTNdAf725aE+mtt92X53l5mbv1871IWONnjyy82MN3i4SAX+jkxdgYahvgBnTjs2WbT0DQ0t5qwcbTAFUFqqyxE83lStBpJw91uTK1WuzxJwY19NRBBdy/sZBxTRb/BVxfpcIjERfQ8KhJWxQkjnTQxPi6Y++ywzNKI5Xs/J/6Qd6OZpROajbqKQUwJ2p7sVHJUNu5Qfx5vvjRPtzCMNogQzbNyMHTlB6r09xAfofvcnLdYt5f/A2JcGesEVhe/gKGh8y1owEAAA==">Web</a>] [<a href="https://frspme.westus2.kusto.windows.net/FRS?query=H4sIAAAAAAAAA5WQTUvDQBCG7/0Vw54SCMZ6kkK8CKI3rd5KkdWMyZbshzMTNdAf725aE+mtt92X53l5mbv1871IWONnjyy82MN3i4SAX+jkxdgYahvgBnTjs2WbT0DQ0t5qwcbTAFUFqqyxE83lStBpJw91uTK1WuzxJwY19NRBBdy/sZBxTRb/BVxfpcIjERfQ8KhJWxQkjnTQxPi6Y++ywzNKI5Xs/J/6Qd6OZpROajbqKQUwJ2p7sVHJUNu5Qfx5vvjRPtzCMNogQzbNyMHTlB6r09xAfofvcnLdYt5f/A2JcGesEVhe/gKGh8y1owEAAA==&web=0">Desktop</a>]</span> [<a href="https://dataexplorer.azure.com/clusters/frspme.westus2/databases/FRS">cluster('frspme.westus2.kusto.windows.net').database('FRS')</a>]<div style="color: #000000;background-color: #fffffe;font-family: Menlo, Monaco, 'Courier New', monospace;font-weight: normal;font-size: 12px;line-height: 18px;white-space: pre;"><div><span style="color: #000000;">FRSHttpRequests</span></div><div><span style="color: #000000;">| where eventTimestamp &gt; ago(1h)</span></div><div><span style="color: #000000;">| where pathCategory == "/deltas/:tenantId/:id"</span></div><div><span style="color: #000000;">|extend url = substring(url, 82)</span></div><div><span style="color: #000000;">| extend queryParameters = parse_json(parse_urlquery(url))</span></div><div><span style="color: #000000;">| extend fromParam = queryParameters["Query Parameters"].["from"]</span></div><div><span style="color: #000000;">| extend toParam = queryParameters["Query Parameters"].["to"]</span></div><div><span style="color: #000000;">| where isempty(fromParam) or isempty(toParam)</span></div><div><span style="color: #000000;">| project eventTimestamp, fromParam, toParam</span></div><div><span style="color: #000000;">| limit 10</span></div></div><div><br/><br/><header>Table0</header>

eventTimestamp | fromParam | toParam
-- | -- | --
2023-07-19T18:12:39.268Z | 11 |  
2023-07-19T18:12:56.675Z | 18 |  
2023-07-19T18:11:05.872Z | 97 |  
2023-07-19T18:11:28.927Z | 14 |  
2023-07-19T18:11:59.882Z | 37 |  
2023-07-19T18:12:35.465Z | 6 |  
2023-07-19T18:12:57.556Z | 8 |  
2023-07-19T18:14:11.809Z | 30 |  
2023-07-19T18:14:22.768Z | 54 |  
2023-07-19T18:14:22.769Z | 154 |  

</div>

The problematic args here could potential causing other scribe flow error here. cc: @pradeepvairamani , @kekachmar